doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc

```diff
@@ -0,0 +1,142 @@
+[id='jws-jdbc-config_{context}']
+= Configuring JDBC Web Server data sources on {JWS}
+
+Java Database Connectivity (JDBC) is an API specification used to connect programs written in Java to the data in popular databases.  A data source is an object that enables a JDBC client, such as an application server, to establish a connection with a database. Applications look up the data source on the Java Naming and Directory Interface (JNDI) tree or in the local application context and request a database connection to retrieve data. You must configure data sources for {KIE_SERVER} to ensure proper data exchange between the servers and the designated database.
```

Comment:
Double space after "databases". Also I would suggest "API specification that is used..." to avoid a double meaning with the other English use of "used to".

Reply:
Done